### ISSUE TITLE
Skip to main content should have a higher color contrast ratio

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org#418]
![image](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiMTRiNDc5ZjMtZWM2YS00ZDBiLWI5NGUtNmZlZmUzMzc4NWUzIiwiZGVwbG95X2lkIjoiNjExZTg5OGE5NDQ4N2IwMDA3NTUxNTAzIiwiaWQiOiI3NGIyN2QyNS03MmE3LTRjOTgtYTkxOC1iZmY1Y2Y4ZWRjZjkifQ==)

### Description of issue
Skip to main content does not meet a11y contrast requirements for a number of colorblind situations. Using a black on white background can solve this easily.

### Expected result
Better contrast ratio
<img width="282" alt="Screen Shot 2021-08-19 at 1 22 22 PM" src="https://user-images.githubusercontent.com/14154792/130114867-f6fc50a8-22fe-48c9-a4b0-74ae06d4bd1a.png">

```
Path:      /
Browser:   Chrome 92.0.4515.131 on Mac OS 10.15.7
Viewport:  1792 x 1009 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-359--dcdesignweek.netlify.app%2F&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=92.0&resolution=1792x1009&speed=1&start=true&ref=netlify-source)

Submitted from [Netlify Deploy Preview #359](https://deploy-preview-359--dcdesignweek.netlify.app/)

**Comment by marcusrelacion** [comment]
@joshkimux this has been fixed.
